index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/font-awesome.min.css"></head>
+    <script src="https://kit.fontawesome.com/7415ea4535.js" crossorigin="anonymous"></script>
+<body>
+
+    <header>
+        <h3>Welcome</h3>
+        <i class="fa-solid fa-bars small"></i>
+    </header>
+
+    <section class="intro">
+        <h2 class="header">I'm <span>Stephen Katuli</span>, nice to meet you</h2>
```

Comment:
- [ ] Please make sure the main title (e.g., "I'm Marianne ...") is an h1 tag. This will help so that If you want that google (and other search engines) know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.